index.html

```diff
@@ -322,7 +322,7 @@ <h1>Grand Prize Winners</h1>
             <span>Developer at FOSSASIA</span>
           </div>
         </div>
-                <div class="col-md-3 col-sm-6 col-md-offset-1 speaker-column">
+                <div class="col-md-3 col-sm-6 speaker-column">
```

Comment:
Don't remove offset! Add the offset in the next speaker column where the author has forgot to add it.

Reply:
#237 Added

Reply:
i wanted to do this.. but that would mean change the entire code in every image down the code with offset. Maybe that wasn't their idea.. So i just changed this in accordance with the way others are alligned.

Reply:
Only change the other grand prize winner not the finalist.

Reply:
ya, I thought the same way that's why I approved your code before. But commonly only 3 winners are there so this might work this way also.

Reply:
> Only change the other grand prize winner not the finalist.

yes, changes only winners no finalists affected.